### ISSUE TITLE
[LowVis] Failures of SC 1.4.3

### ISSUE THREAD

**Issue by goetsu** [w3c/wcag#94]
I don't understand why there is still no failure against text contrast ratio and I think we also need to ask for at least a 3.0:1 ratio for icons used without text. 
It's a more and more common issue especially with responsive website and social sharing. 

There is already a technique for that "Making icons using simple line drawings that meet the contrast provisions for text (future link) " but in advisory techniques for now.

So I think we need two new failures like : 

Failure of Success Criterion 1.4.3 and 1.4.6 due to using icons that do not provide sufficient contrast with background
Failure of Success Criterion 1.4.3 and 1.4.6 due to using text that do not provide sufficient contrast with background

**Comment by awkawk** [comment]
@goetsu - There is a failure  for this, F83 (http://www.w3.org/TR/2010/NOTE-WCAG20-TECHS-20101014/F83.html) which refers to text over background images and is therefore a more complicated case than just text over a colored background. We will need to take a look at that and either modify this technique or add a new one - if you would like to write up the technique that would certainly help get it going.

For icons without text, we don't have a technique because the WCAG success criteria don't at this time require specific contrast ratios for non-text-containing icons.  We responded to a related comment  last August (https://lists.w3.org/Archives/Public/public-comments-wcag20/2014Aug/0022.html) and made an addition to the Understanding document for 1.4.3 to indicate that while the success criteria don't require contrast in images without text it is a good idea to ensure that good contrast is provided (http://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html).

The idea of icons needing to conform to contrast requirements is already listed on our "post WCAG 2.0" comments page (https://www.w3.org/WAI/GL/wiki/Post_WCAG_2#Contrast_in_Icons), so we will keep it in mind as a possible future requirement.

**Comment by goetsu** [comment]
thanks for this answer, I will see what I can do for the technique. For the contrast of icons maybe  creating the technique marked as futur link could help

**Comment by goetsu** [comment]
done using http://www.w3.org/WAI/GL/WCAG20/TECHS-SUBMIT/ for text color of background color

**Comment by michael-n-cooper** [comment]
Closing this issue, the submitted technique will be processed in issue 95

**Comment by mraccess77** [comment]
When does a icon stop being an image of text and start becoming an image that is not text?  What is text may change depending on the language  Is a single character text such as an "f" for Facebook or "g" for Google?
